### PR TITLE
Feat/2079 tsv

### PR DESCRIPTION
**Pull request by alex-wilmer**
TODO:
- [x] format the fields correctly before sending to special endpoint (send facets along etc)

### CODE REVIEW

**Comment by alex-wilmer** [comment]
Projects will not work yet until the endpoint is created on the API. I would prefer this gets looked at / tested before I bother with that.

**Comment by alex-wilmer** [comment]
Added projects support

**Comment by alex-wilmer** [comment]
api link https://github.com/NCI-GDC/gdcapi/pull/546

File: app/scripts/components/tables/export-table.service.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      var model = this.getModel(endpoint);
+
+      var tsvHeadings = headings.reduce((acc, x) => ([
+        ...acc,
+        ...(!x.hidden && x.isField
+          ? x.id === `summary.data_categories`
+            ? Object.keys(this.DATA_CATEGORIES).map(k =>
+              `${this.DATA_CATEGORIES[k].abbr}~~~summary.data_categories.${this.DATA_CATEGORIES[k].full}`
+            )
+            : [`${x.name}~~~${x.id}`]
```
Comment (shanewilson) [inline app/scripts/components/tables/export-table.service.ts]: why do we need this?

Comment (alex-wilmer) [inline app/scripts/components/tables/export-table.service.ts]: I think I can fix this now that I know better how flask handles formdata

Comment (alex-wilmer) [inline app/scripts/components/tables/export-table.service.ts]: It was to send an array of key value pairs

**Comment by alex-wilmer** [comment]
@shanewilson @cy feedback addressed.. using proper form data now, but random squigglies for key value pairs

**Pull request closed without merging**